src/ex6/client/src/App.css

```diff
@@ -0,0 +1,5 @@
+@import "styles/app.css";
+@import "styles/common.css";
+@import "styles/todo.css";
```

Comment:
should be imported in the component js file

Reply:
done